src/utils.cpp

```diff
@@ -190,6 +193,9 @@ TextChat* TextChat::unserialize(class MegaClient *client, string *d)
     ou = MemAccess::get<handle>(ptr);
     ptr += sizeof ou;
 
+    ts = (uint32_t)MemAccess::get<time_t>(ptr);
```

Comment:
Yes, definitely. I was influenced by the datatype used internally by the chat library to represent this timestamp. I'll fix that.